core/modules/SConscript

```diff
@@ -104,13 +112,23 @@ def standardModule(env, exclude=None, module=None, test_libs=None, unit_tests=No
         module = os.path.basename(os.getcwd())
     state.log.debug('standardModule: module = %s path = %s' % (module, path))
 
+    # using two optional lists provided by the nmodule to build names of
+    # *.cc files to be excluded from being used to build module's
+    # library or unit tests
+    exclude_cc_files = []
+    if exclude is not None:
+        exclude_cc_files = exclude
```

Comment:
`exclude_cc_files += exclude`, otherwise you are potentially modifying `exclude` too

Reply:
Agreed. My Python is rusty :-)